services/metadata_service/api/utils.py

```diff
@@ -53,10 +54,20 @@ def handle_exceptions(func):
     """Catch exceptions and return appropriate HTTP error."""
 
     @wraps(func)
-    async def wrapper(*args, **kwargs):
+    async def wrapper(self, request):
         try:
-            return await func(*args, **kwargs)
+            return await func(self, request)
         except Exception as err:
-            return http_500(str(err))
+            # either use provided traceback from subprocess, or generate trace from current process
+            err_trace = getattr(err, 'traceback_str', None) or get_traceback_str()
+            print(err_trace)
```

Comment:
Necessary print or leftover from some testing? If this is for logging purposes, consider using the logger from `services.utils` instead. 

If this is for generic logging over the process, it should probably be set as a logging handler with `loop.set_exception_handler(some_error_handling_function)` in the server setup phase instead. You can see https://github.com/Netflix/metaflow-service/blob/ui/services/ui_backend_service/ui_server.py#L98 for an example

Reply:
I will use the logger. I did want to log things to aid in debugging after.